### DISCUSSION TITLE
校内学习

### DISCUSSION THREAD

**Discussion by langningchen** [XMOJ-Script-dev/XMOJ-Script#270] [General]
@PythonSmall-Q 你们数学校内学到哪里了？

**Comment by langningchen** [comment]
@boomzero And you?

**Comment by PythonSmall-Q** [comment]
我们圆学完了

**Comment by PythonSmall-Q** [comment]
@langningchen

**Comment by boomzero** [comment]
我们统计学完了，但是没学圆

**Comment by PythonSmall-Q** [comment]
统计看空中课堂不几句可以了吗

**Comment by boomzero** [comment]
……

**Comment by langningchen** [comment]
你们都学得好快啊

**Comment by langningchen** [comment]
我们才刚学完二次函数

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 正在刷一模卷

**Comment by boomzero** [comment]
+1

**Comment by boomzero** [comment]
我们已经在一轮复习了

**Comment by langningchen** [comment]
是的，我们也在第一轮复习

**Comment by langningchen** [comment]
@boomzero @PythonSmall-Q 你们也是今天一模吧？考的怎么样？你们两个区的卷子难吗？我们浦东的感觉这次不太简单

**Comment by boomzero** [comment]
你们区不是用闵行卷吗

**Reply by boomzero** [reply]
至少,上海中学东校用的是闵行卷

**Comment by boomzero** [comment]
评价一下,我考理化时体温38.7

**Comment by boomzero** [comment]
<img width="820" alt="image" src="https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/85378277/d1d40bf0-64e6-464d-bce2-30ec47d79219">
闵行语文

**Comment by boomzero** [comment]
@langningchen

**Comment by langningchen** [comment]
@boomzero 为什么你们全都说我用的闵行卷？我们学习用的自己的浦东卷，浦东有自己的卷子

**Comment by langningchen** [comment]
我们作文题目是“我，被看到了”

**Comment by langningchen** [comment]
@boomzero 数学保佑

**Comment by boomzero** [comment]
> 至少,上海中学东校用的是闵行卷

**Comment by boomzero** [comment]
> @boomzero 为什么你们全都说我用的闵行卷？我们学习用的自己的浦东卷，浦东有自己的卷子

但不是泄题了吗

**Comment by boomzero** [comment]
> @boomzero 数学保佑

emm

**Comment by langningchen** [comment]
> > @boomzero 为什么你们全都说我用的闵行卷？我们学习用的自己的浦东卷，浦东有自己的卷子
> 
> 但不是泄题了吗

泄题？

**Comment by langningchen** [comment]
@PythonSmall-Q 你一模成绩怎么样？

**Comment by langningchen** [comment]
@boomzero 还有你呢？

**Comment by boomzero** [comment]
Umm

**Reply by langningchen** [reply]
@boomzero 你呢？不想说吗？

**Comment by PythonSmall-Q** [comment]
语文122，数学132，英语135，物理58（脑子坏了），化学46（脑子坏了），道法30

**Reply by PythonSmall-Q** [reply]
@langningchen

**Reply by langningchen** [reply]
@PythonSmall-Q 你竟然也考得这么好

**Reply by langningchen** [reply]
我快疯了，我就算便问了5个人，全都比我高 :-1:

**Reply by langningchen** [reply]
不过我跟你的成绩没有可比性，因为根本不在一个区

**Reply by PythonSmall-Q** [reply]
md我3%，年纪24

**Reply by PythonSmall-Q** [reply]
2%-3%是区里的

**Reply by langningchen** [reply]
我就很逊了，才 7.1%

**Comment by PythonSmall-Q** [comment]
![image](https://github.com/XMOJ-Script-dev/XMOJ-Script/assets/106425289/25889683-31ee-481c-bf17-7641ff5f05af)
 @langningchen @boomzero 有人误点了退出让我把她拉回来，这体现了2fa的重要性

**Reply by PythonSmall-Q** [reply]
还有passkey

**Reply by langningchen** [reply]
@PythonSmall-Q “她”？

**Reply by PythonSmall-Q** [reply]
他

**Comment by boomzero** [comment]
Wait a sec

**Reply by langningchen** [reply]
这已经过去了多少秒了？

**Reply by boomzero** [reply]
well, I didn't have time that day

**Comment by PythonSmall-Q** [comment]
我妈说我越来越唐了，我们同学说我越来越茶了…………………………

**Reply by langningchen** [reply]
？

**Reply by boomzero** [reply]
?

**Comment by boomzero** [comment]
tmd, 14%

**Reply by langningchen** [reply]
那说明你们学校好

**Reply by langningchen** [reply]
尽管你考得很好才能排到 14%

**Reply by boomzero** [reply]
md年级58

**Reply by boomzero** [reply]
是初三考的最烂的一次

**Comment by PythonSmall-Q** [comment]
aya没事的，黄浦区很菜的

**Reply by langningchen** [reply]
@PythonSmall-Q 你竟然歧视黄浦区，难不成你是杨浦的？

**Comment by boomzero** [comment]
很好...

**Comment by boomzero** [comment]
swx不是黄浦的吗

**Comment by boomzero** [comment]
OK, let me tell you

115+124+140.5+58+43+10+24=514.5

(怎么说呢, 我考试时38.7度)

**Reply by boomzero** [reply]
(但是我语文作文54)

**Comment by PythonSmall-Q** [comment]
作文47，写的最烂的一次

**Comment by langningchen** [comment]
作文 51
